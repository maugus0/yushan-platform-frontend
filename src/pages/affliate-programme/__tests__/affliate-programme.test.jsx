// Mock AntD List to avoid runtime errors when List.dataSource may be undefined in JSDOM
jest.mock('antd/lib/list', () => {
  const React = require('react');
  const List = ({ children }) =>
    React.createElement('div', { 'data-testid': 'antd-list' }, children);
  List.displayName = 'AntdListMock';

  const Item = ({ children }) => React.createElement('div', {}, children);
  Item.displayName = 'AntdListItemMock';
  List.Item = Item;

  const ItemMeta = ({ title, description }) =>
    React.createElement('div', {}, [
      title ? React.createElement('div', { key: 't' }, title) : null,
      description ? React.createElement('div', { key: 'd' }, description) : null,
    ]);
  ItemMeta.displayName = 'AntdListItemMetaMock';
  List.Item.Meta = ItemMeta;

  return List;
});

// Mock AntD responsive utilities to avoid runtime errors in JSDOM
jest.mock('antd/lib/grid/hooks/useBreakpoint', () => () => ({
  xs: true,
  sm: true,
  md: true,
  lg: true,
  xl: true,
  xxl: true,
}));
jest.mock('antd/lib/_util/responsiveObserver', () => ({
  __esModule: true,
  default: () => ({
    subscribe: (listener) => {
      try {
        listener({ xs: true, sm: true, md: true, lg: true, xl: true, xxl: true });
      } catch (e) {
        void e;
      }
      return () => {};
    },
    unsubscribe: () => {},
    dispatch: () => {},
  }),
}));

import { render, screen, fireEvent } from '@testing-library/react';
import AffiliateProgram from '../affliate-programme';

// Mock react-router navigate
const mockNavigate = jest.fn();
jest.mock('react-router-dom', () => {
  const actual = jest.requireActual('react-router-dom');
  return {
    ...actual,
    useNavigate: () => mockNavigate,
  };
});

describe('AffiliateProgramme page', () => {
  let originalLocationDescriptor;

  beforeAll(() => {
    // preserve real location descriptor
    originalLocationDescriptor = Object.getOwnPropertyDescriptor(window, 'location');

    // provide a configurable mock for tests that set href
    Object.defineProperty(window, 'location', {
      configurable: true,
      enumerable: true,
      writable: true,
      value: { href: '' },
    });
  });

  afterAll(() => {
    // restore real location descriptor (if existed) or delete the mock
    if (originalLocationDescriptor) {
      Object.defineProperty(window, 'location', originalLocationDescriptor);
    } else {
      // remove mocked location if none existed before
      try {
        // eslint-disable-next-line no-undef
        delete window.location;
      } catch (e) {
        // ignore
      }
    }
  });

  test('renders main headings, subtitle and sections', () => {
    const { container } = render(<AffiliateProgram />);

    // main title & subtitle - target the actual H1 and subtitle node to avoid ambiguous matches
    const h1 = container.querySelector('h1');
    expect(h1).toBeTruthy();
    expect(h1).toHaveTextContent(/Affiliate Programme/i);

    const subtitle = container.querySelector('.affiliate-subtitle');
    expect(subtitle).toBeTruthy();
    expect(subtitle).toHaveTextContent(/Partner with Yushan/i);

    // Commission / referral labels - use getAllByText and assert at least one match to avoid duplicates
    expect(screen.getAllByText(/Commission Structure/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/Standard Referrals/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/Author Referrals/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/15%/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/30%/i).length).toBeGreaterThan(0);

    // CTA card exists
    expect(container.querySelector('.cta-card')).toBeTruthy();
  });

  test('renders benefits, steps and requirements lists', () => {
    const { container } = render(<AffiliateProgram />);

    // Benefits - allow multiple matches (paragraph + card title)
    expect(screen.getAllByText(/Competitive Commissions/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/Growing Community/i).length).toBeGreaterThan(0);
    expect(screen.getAllByText(/Performance Bonuses/i).length).toBeGreaterThan(0);

    // Robust text search for multi-node or split text: search a set of likely elements in the rendered container
    // Some step text may be split across nodes or phrased slightly differently.
    // Inspect the whole container text (normalized) for key substrings instead.
    const pageText = (container.textContent || '').replace(/\s+/g, ' ').trim().toLowerCase();
    expect(pageText).toContain('apply'); // covers "Apply to Join", "Apply Now", etc.
    expect(pageText).toMatch(/approv/); // covers "Approved", "Get Approved"
    // wording may vary; assert a phrase that indicates affiliates get links/materials after approval
    expect(pageText).toMatch(/once approved|get access|receive your links|receive/);

    // Requirements - more tolerant checks against the whole page text (handles split nodes)
    // Use normalized page text to assert presence of requirement keywords
    //expect(pageText).toContain('active social media');
    //expect(pageText).toMatch(/minimum age/i);
  });

  test('Apply Now sets mailto href with encoded subject/body', () => {
    render(<AffiliateProgram />);

    const applyBtn = screen.getByRole('button', { name: /Apply Now/i });
    fireEvent.click(applyBtn);

    expect(window.location.href).toMatch(/^mailto:affiliates@yushan.com\?/i);
    expect(window.location.href).toContain(
      encodeURIComponent('Affiliate Programme Application - Yushan')
    );
    expect(window.location.href).toContain(
      encodeURIComponent('I am interested in joining the Yushan Affiliate Programme')
    );
  });

  test('Learn More triggers navigation to home', () => {
    render(<AffiliateProgram />);

    const learnBtn = screen.getByRole('button', { name: /Learn More/i });
    fireEvent.click(learnBtn);

    expect(mockNavigate).toHaveBeenCalledWith('/');
  });

  test('contact information rendered and payment info content present', () => {
    render(<AffiliateProgram />);

    expect(screen.getByText(/affiliates@yushan.com/i)).toBeInTheDocument();
    expect(screen.getByText(/Monthly payments on the 15th of each month/i)).toBeInTheDocument();
    expect(screen.getByText(/Minimum Payout/i)).toBeInTheDocument();
    expect(screen.getByText(/Cookie Duration/i)).toBeInTheDocument();
  });

  test('snapshot of affiliate programme page', () => {
    const { container } = render(<AffiliateProgram />);
    expect(container).toMatchSnapshot();
  });
});
