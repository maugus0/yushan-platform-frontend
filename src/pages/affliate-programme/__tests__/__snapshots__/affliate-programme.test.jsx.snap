// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AffiliateProgramme page snapshot of affiliate programme page 1`] = `
<div>
  <div
    class="affiliate-page"
  >
    <div
      class="affiliate-container"
    >
      <div
        class="ant-row ant-row-center css-dev-only-do-not-override-1odpy5d"
      >
        <div
          class="ant-col ant-col-xs-24 ant-col-sm-22 ant-col-md-20 ant-col-lg-18 ant-col-xl-16 css-dev-only-do-not-override-1odpy5d"
        >
          <div
            class="ant-card ant-card-bordered affiliate-card css-dev-only-do-not-override-1odpy5d"
          >
            <div
              class="ant-card-body"
            >
              <div
                class="affiliate-header"
              >
                <h1
                  class="ant-typography affiliate-title css-dev-only-do-not-override-1odpy5d"
                >
                  Affiliate Programme
                </h1>
                <div
                  class="ant-typography affiliate-subtitle css-dev-only-do-not-override-1odpy5d"
                >
                  Partner with Yushan and earn while helping readers discover amazing stories
                </div>
              </div>
              <div
                class="ant-divider css-dev-only-do-not-override-1odpy5d ant-divider-horizontal"
                role="separator"
              />
              <div
                class="affiliate-content"
              >
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Why Join Our Affiliate Programme?
                </h2>
                <div
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Yushan's Affiliate Programme offers content creators, bloggers, and social media influencers the opportunity to earn competitive commissions by promoting our platform to their audience. Help readers discover their next favorite story while building a sustainable income stream.
                </div>
                <div
                  class="benefits-section"
                >
                  <div
                    class="ant-row css-dev-only-do-not-override-1odpy5d"
                    style="margin-left: -12px; margin-right: -12px; row-gap: 24px;"
                  >
                    <div
                      class="ant-col ant-col-xs-24 ant-col-md-8 css-dev-only-do-not-override-1odpy5d"
                      style="padding-left: 12px; padding-right: 12px;"
                    >
                      <div
                        class="ant-card ant-card-bordered ant-card-hoverable benefit-card css-dev-only-do-not-override-1odpy5d"
                      >
                        <div
                          class="ant-card-body"
                        >
                          <div
                            class="benefit-icon"
                          >
                            <span
                              aria-label="dollar"
                              class="anticon anticon-dollar"
                              role="img"
                              style="font-size: 32px; color: rgb(82, 196, 26);"
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="dollar"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372zm47.7-395.2l-25.4-5.9V348.6c38 5.2 61.5 29 65.5 58.2.5 4 3.9 6.9 7.9 6.9h44.9c4.7 0 8.4-4.1 8-8.8-6.1-62.3-57.4-102.3-125.9-109.2V263c0-4.4-3.6-8-8-8h-28.1c-4.4 0-8 3.6-8 8v33c-70.8 6.9-126.2 46-126.2 119 0 67.6 49.8 100.2 102.1 112.7l24.7 6.3v142.7c-44.2-5.9-69-29.5-74.1-61.3-.6-3.8-4-6.6-7.9-6.6H363c-4.7 0-8.4 4-8 8.7 4.5 55 46.2 105.6 135.2 112.1V761c0 4.4 3.6 8 8 8h28.4c4.4 0 8-3.6 8-8.1l-.2-31.7c78.3-6.9 134.3-48.8 134.3-124-.1-69.4-44.2-100.4-109-116.4zm-68.6-16.2c-5.6-1.6-10.3-3.1-15-5-33.8-12.2-49.5-31.9-49.5-57.3 0-36.3 27.5-57 64.5-61.7v124zM534.3 677V543.3c3.1.9 5.9 1.6 8.8 2.2 47.3 14.4 63.2 34.4 63.2 65.1 0 39.1-29.4 62.6-72 66.4z"
                                />
                              </svg>
                            </span>
                          </div>
                          <h4
                            class="ant-typography benefit-title css-dev-only-do-not-override-1odpy5d"
                          >
                            Competitive Commissions
                          </h4>
                          <div
                            class="ant-typography benefit-description css-dev-only-do-not-override-1odpy5d"
                          >
                            Earn up to 30% commission on all successful referrals and premium subscriptions.
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="ant-col ant-col-xs-24 ant-col-md-8 css-dev-only-do-not-override-1odpy5d"
                      style="padding-left: 12px; padding-right: 12px;"
                    >
                      <div
                        class="ant-card ant-card-bordered ant-card-hoverable benefit-card css-dev-only-do-not-override-1odpy5d"
                      >
                        <div
                          class="ant-card-body"
                        >
                          <div
                            class="benefit-icon"
                          >
                            <span
                              aria-label="team"
                              class="anticon anticon-team"
                              role="img"
                              style="font-size: 32px; color: rgb(24, 144, 255);"
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="team"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M824.2 699.9a301.55 301.55 0 00-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 00-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 008 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 01612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 008-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 01-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 01612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 01-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 008 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
                                />
                              </svg>
                            </span>
                          </div>
                          <h4
                            class="ant-typography benefit-title css-dev-only-do-not-override-1odpy5d"
                          >
                            Growing Community
                          </h4>
                          <div
                            class="ant-typography benefit-description css-dev-only-do-not-override-1odpy5d"
                          >
                            Join a thriving community of content creators and earn from our expanding user base.
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="ant-col ant-col-xs-24 ant-col-md-8 css-dev-only-do-not-override-1odpy5d"
                      style="padding-left: 12px; padding-right: 12px;"
                    >
                      <div
                        class="ant-card ant-card-bordered ant-card-hoverable benefit-card css-dev-only-do-not-override-1odpy5d"
                      >
                        <div
                          class="ant-card-body"
                        >
                          <div
                            class="benefit-icon"
                          >
                            <span
                              aria-label="trophy"
                              class="anticon anticon-trophy"
                              role="img"
                              style="font-size: 32px; color: rgb(250, 173, 20);"
                            >
                              <svg
                                aria-hidden="true"
                                data-icon="trophy"
                                fill="currentColor"
                                focusable="false"
                                height="1em"
                                viewBox="64 64 896 896"
                                width="1em"
                              >
                                <path
                                  d="M868 160h-92v-40c0-4.4-3.6-8-8-8H256c-4.4 0-8 3.6-8 8v40h-92a44 44 0 00-44 44v148c0 81.7 60 149.6 138.2 162C265.7 630.2 359 721.7 476 734.5v105.2H280c-17.7 0-32 14.3-32 32V904c0 4.4 3.6 8 8 8h512c4.4 0 8-3.6 8-8v-32.3c0-17.7-14.3-32-32-32H548V734.5C665 721.7 758.3 630.2 773.8 514 852 501.6 912 433.7 912 352V204a44 44 0 00-44-44zM184 352V232h64v207.6a91.99 91.99 0 01-64-87.6zm520 128c0 49.1-19.1 95.4-53.9 130.1-34.8 34.8-81 53.9-130.1 53.9h-16c-49.1 0-95.4-19.1-130.1-53.9-34.8-34.8-53.9-81-53.9-130.1V184h384v296zm136-128c0 41-26.9 75.8-64 87.6V232h64v120z"
                                />
                              </svg>
                            </span>
                          </div>
                          <h4
                            class="ant-typography benefit-title css-dev-only-do-not-override-1odpy5d"
                          >
                            Performance Bonuses
                          </h4>
                          <div
                            class="ant-typography benefit-description css-dev-only-do-not-override-1odpy5d"
                          >
                            Get additional bonuses for reaching monthly and yearly performance milestones.
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Commission Structure
                </h2>
                <div
                  class="commission-info"
                >
                  <div
                    class="ant-row css-dev-only-do-not-override-1odpy5d"
                    style="margin-left: -12px; margin-right: -12px; row-gap: 24px;"
                  >
                    <div
                      class="ant-col ant-col-xs-24 ant-col-md-12 css-dev-only-do-not-override-1odpy5d"
                      style="padding-left: 12px; padding-right: 12px;"
                    >
                      <div
                        class="ant-card ant-card-bordered commission-card css-dev-only-do-not-override-1odpy5d"
                      >
                        <div
                          class="ant-card-body"
                        >
                          <h4
                            class="ant-typography css-dev-only-do-not-override-1odpy5d"
                          >
                            Standard Referrals
                          </h4>
                          <div
                            class="commission-rate"
                          >
                            15%
                          </div>
                          <div
                            class="ant-typography css-dev-only-do-not-override-1odpy5d"
                          >
                            Earn 15% commission on all premium subscriptions from your referrals
                          </div>
                        </div>
                      </div>
                    </div>
                    <div
                      class="ant-col ant-col-xs-24 ant-col-md-12 css-dev-only-do-not-override-1odpy5d"
                      style="padding-left: 12px; padding-right: 12px;"
                    >
                      <div
                        class="ant-card ant-card-bordered commission-card css-dev-only-do-not-override-1odpy5d"
                      >
                        <div
                          class="ant-card-body"
                        >
                          <h4
                            class="ant-typography css-dev-only-do-not-override-1odpy5d"
                          >
                            Author Referrals
                          </h4>
                          <div
                            class="commission-rate"
                          >
                            30%
                          </div>
                          <div
                            class="ant-typography css-dev-only-do-not-override-1odpy5d"
                          >
                            Earn 30% commission when you refer successful authors to our platform
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  How It Works
                </h2>
                <div
                  data-testid="antd-list"
                />
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Requirements
                </h2>
                <div
                  data-testid="antd-list"
                />
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Payment Information
                </h2>
                <div
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Payment Schedule:
                    </strong>
                  </span>
                   Monthly payments on the 15th of each month
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Minimum Payout:
                    </strong>
                  </span>
                   $50 USD
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Payment Methods:
                    </strong>
                  </span>
                   Bank transfer, PayPal, or digital wallets
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Cookie Duration:
                    </strong>
                  </span>
                   30 days attribution window
                </div>
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Marketing Materials
                </h2>
                <div
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Once approved, you'll get access to:
                </div>
                <ul
                  class="materials-list"
                >
                  <li>
                    Branded banners and graphics in various sizes
                  </li>
                  <li>
                    Product screenshots and promotional images
                  </li>
                  <li>
                    Email templates and social media content
                  </li>
                  <li>
                    Video promotional materials
                  </li>
                  <li>
                    Real-time tracking dashboard
                  </li>
                </ul>
                <div
                  class="cta-section"
                >
                  <div
                    class="ant-card ant-card-bordered cta-card css-dev-only-do-not-override-1odpy5d"
                  >
                    <div
                      class="ant-card-body"
                    >
                      <h3
                        class="ant-typography css-dev-only-do-not-override-1odpy5d"
                      >
                        Ready to Get Started?
                      </h3>
                      <div
                        class="ant-typography css-dev-only-do-not-override-1odpy5d"
                      >
                        Join thousands of affiliates already earning with Yushan. Apply today and start monetizing your audience while promoting quality content.
                      </div>
                      <div
                        class="ant-space css-dev-only-do-not-override-1odpy5d ant-space-horizontal ant-space-align-center ant-space-gap-row-middle ant-space-gap-col-middle"
                      >
                        <div
                          class="ant-space-item"
                        >
                          <button
                            class="ant-btn css-dev-only-do-not-override-1odpy5d ant-btn-primary ant-btn-color-primary ant-btn-variant-solid ant-btn-lg"
                            type="button"
                          >
                            <span
                              class="ant-btn-icon"
                            >
                              <span
                                aria-label="mail"
                                class="anticon anticon-mail"
                                role="img"
                              >
                                <svg
                                  aria-hidden="true"
                                  data-icon="mail"
                                  fill="currentColor"
                                  focusable="false"
                                  height="1em"
                                  viewBox="64 64 896 896"
                                  width="1em"
                                >
                                  <path
                                    d="M928 160H96c-17.7 0-32 14.3-32 32v640c0 17.7 14.3 32 32 32h832c17.7 0 32-14.3 32-32V192c0-17.7-14.3-32-32-32zm-40 110.8V792H136V270.8l-27.6-21.5 39.3-50.5 42.8 33.3h643.1l42.8-33.3 39.3 50.5-27.7 21.5zM833.6 232L512 482 190.4 232l-42.8-33.3-39.3 50.5 27.6 21.5 341.6 265.6a55.99 55.99 0 0068.7 0L888 270.8l27.6-21.5-39.3-50.5-42.7 33.2z"
                                  />
                                </svg>
                              </span>
                            </span>
                            <span>
                              Apply Now
                            </span>
                          </button>
                        </div>
                        <div
                          class="ant-space-item"
                        >
                          <button
                            class="ant-btn css-dev-only-do-not-override-1odpy5d ant-btn-default ant-btn-color-default ant-btn-variant-outlined ant-btn-lg"
                            type="button"
                          >
                            <span>
                              Learn More
                            </span>
                          </button>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
                <h2
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Contact Our Affiliate Team
                </h2>
                <div
                  class="ant-typography css-dev-only-do-not-override-1odpy5d"
                >
                  Have questions about our affiliate programme? Our dedicated affiliate team is here to help:
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Email:
                    </strong>
                  </span>
                   affiliates@yushan.com
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Response Time:
                    </strong>
                  </span>
                   Within 24 hours
                  <br />
                  <span
                    class="ant-typography css-dev-only-do-not-override-1odpy5d"
                  >
                    <strong>
                      Support Hours:
                    </strong>
                  </span>
                   Monday - Friday, 9 AM - 6 PM SGT
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
